site/ar.njk

```diff
@@ -0,0 +1,149 @@
+---
+layout: default
+meta_title: نداء للحد من جائحة كوفيد-١٩
+meta_description: الحكومات لن تستطيع بمفردها حل المشكلة. على كل فرد منّا أن يقوم بواجبه. الله يخليك خليك بالبيت
```

Comment:
this is Egyptian dialect

Reply:
It is actually a reference to a popular folk song by the Lebanese singer Fairuz (https://en.wikipedia.org/wiki/Fairuz) that is now being used as a hashtag to encourage self-quarantine across the Arab world: https://twitter.com/hashtag/%D8%AE%D9%84%D9%8A%D9%83_%D8%A8%D8%A7%D9%84%D8%A8%D9%8A%D8%AA

I saw that other language translations maintained the original English hashtag #StayTheFuckHome where the hashtag was used, and figured it is best to keep using the site's original version for consistency.

Reply:
great! I like it.